src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -128,7 +178,12 @@ async function boot(options?: Partial<WebAssemblyStartOptions>): Promise<void> {
   }
 
   // Start up the application
-  platform.callEntryPoint(resourceLoader.bootConfig.entryAssembly);
+  invokeBlazorStartedCallbacks(platform.callEntryPoint(resourceLoader.bootConfig.entryAssembly));
+
+  async function invokeBlazorStartedCallbacks(applicationStarted: Promise<void>) {
+    await applicationStarted;
+    await Promise.all(afterBlazorStartedCallbacks.map(c => c()));
+  }
```

Comment:
Not sure why this is a function - can it be inlined into the code above?

Reply:
Oh is it because you don't want the returned promise to include these awaits?

Reply:
Yep